src/persist/src/indexed/mod.rs

```diff
@@ -121,13 +121,8 @@ impl<K: Data, V: Data, U: Buffer, L: Blob> Indexed<K, V, U, L> {
     }
 
     fn new_blob_key(&mut self) -> String {
-        // TODO: Use meaningful file names? Something like id+desc might be
```

Comment:
This is going to get complicated enough that I don't think this gets us anything debug-ability-wise